controllers/core/openstackcontrolplane_controller.go

```diff
@@ -84,6 +85,8 @@ type OpenStackControlPlaneReconciler struct {
 //+kubebuilder:rbac:groups=neutron.openstack.org,resources=neutronapis,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=ovn.openstack.org,resources=ovndbclusters,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=ovn.openstack.org,resources=ovnnorthds,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=ovs.openstack.org,resources=ovs,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=ovs.openstack.org,resources=ovs/status,verbs=get;update;patch
```

Comment:
Does the OpenStack operator actually update the OVS CR status?  It doesn't seem like it does.  And I don't think OVS CR status should be updated outside of the OVS operator anyhow, as per best practices (perhaps).  We can probably remove this annotation.

Reply:
Done